Sources_v3/StreamChatUI/ChatChannel/MessageComposerInputAccessoryViewController.swift

```diff
@@ -35,15 +33,12 @@ open class MessageComposerInputAccessoryViewController<ExtraData: ExtraDataTypes
         }
     }
     
-    public var replyMessage: _ChatMessage<ExtraData>? {
+    var isEmpty: Bool = true {
         didSet {
-            state = .reply
-        }
-    }
-    
-    public var messageToEdit: _ChatMessage<ExtraData>? {
-        didSet {
-            state = .edit
+            composerView.attachmentButton.isHidden = !isEmpty
+            composerView.commandsButton.isHidden = !isEmpty
```

Comment:
You can use isVisible 🤯

Reply:
TBH, I found `isVisible` API confusing a bit cause it means some of the properties we set will be `isHidden` and some will be `isVisible`. I feel like it will take more time for me to understand from the glance what is happening here. Or the idea is to use `isVisible` all the time?